sphinx_automodapi/automodapi.py

```diff
@@ -93,6 +93,12 @@
     Should be a bool and if ``True`` members that a class inherits from a base
     class are included in the generated documentation. Defaults to ``False``.
 
+* ``automodsumm_included_members``
+    A list of strings containing the names of hidden class members that should be
+    included in the documentation. This is most commonly used to add special class
+    methods like ``__getitem__`` and ``__setitem__``. Defaults to
+    ``['__init__', '__call__']``.
+
```

Comment:
It's a bit confusing to document this option in the `automodapi` module. Some options are documented in both modules, some other are not ...